Seeing this locally with `make setup-cluster`:


```
docker buildx build  --load --progress=auto -t us-central1-docker.pkg.dev/k8s-staging-images/agentic-net/agentic-networking-controller:main -t us-central1-docker.pkg.dev/k8s-staging-images/agentic-net/agentic-networking-controller:feat/setup-cluster-target --pull --build-arg GO_VERSION=1.26.0 --label runnumber=0 .
[+] Building 0.0s (0/0)                                                                                                                                                                               docker:orbstack
ERROR: failed to build: invalid tag "us-central1-docker.pkg.dev/k8s-staging-images/agentic-net/agentic-networking-controller:feat/setup-cluster-target": invalid reference format
make[1]: *** [build-all] Error 1
make: *** [setup-cluster] Error 2
```

The build fails on branches with `/` in the name (like `feat/setup-cluster-target`) because `BASE_REF` in `hack/build/Makefile.common.in:26` defaults to `git rev-parse --abbrev-ref HEAD`, which gets used as a Docker tag — and Docker tags can't contain `/`.

```
ERROR: failed to build: invalid tag "...agentic-networking-controller:feat/setup-cluster-target": invalid reference format
```

Pre-existing issue in `Makefile.common.in`, but this PR surfaces it directly since the primary use case is running from a feature branch.

One fix — sanitize the default in `Makefile.common.in:26`:

```makefile
BASE_REF ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null | tr '/' '-' || echo latest)
```

Could be done in this PR or a separate one.

--- a/hack/build/Makefile.common.in
+++ b/hack/build/Makefile.common.in
@@ -23,7 +23,7 @@ REPO_ROOT?=$(abspath $(dir $(lastword $(MAKEFILE_LIST)))/../..)
 
 # Default tags for local builds
 TAG ?= $(shell git describe --tags --always --dirty 2>/dev/null || echo latest)
-BASE_REF ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null || echo latest)
+BASE_REF ?= $(shell git rev-parse --abbrev-ref HEAD 2>/dev/null | tr '/' '-' || echo latest)
 
 # Go version to use, respected by images that build go binaries
 GO_VERSION?=$(shell cat $(REPO_ROOT)/.go-version | head -n1)

